src/pretix/control/templates/pretixcontrol/event/plugins.html

```diff
@@ -6,8 +6,9 @@
     <h1>{% trans "Available plugins" %}</h1>
     <p>
         {% blocktrans trimmed %}
-            On this page, you can choose plugins you want to enable for your event. Plugins might bring additional
-            software functionality, connect your event to third-party services, or apply other forms of customizations.
+            On this page, you can select the plugins you would like to enable for your event. These plugins can enhance
+            software functionality, integrate your event with third-party services, or provide other customization
+            options.
```

Comment:
On this page, you can select the plugins you would like to enable for your event. Plugins can add extra functionality, integrate third-party services, or provide various customization options to enhance your event experience.